lib/puppet/indirector/catalog/compiler.rb

```diff
@@ -89,15 +89,9 @@ def compile(node)
 
     benchmark(:notice, str) do
       Puppet::Util::Profiler.profile(str, [:compiler, :compile, node.environment, node.name]) do
-        begin
-          config = Puppet::Parser::Compiler.compile(node)
-        rescue Puppet::Error => detail
-          Puppet.err(detail.to_s) if networked?
-          raise
-        end
```

Comment:
where is the above done now? (seems like it logged only if networked (an agent request? or something else?). understand that new version does not catch anything (better), but want to make sure that the implication of not logging if networking is clear.